compat/win32/pthread.c

```diff
@@ -57,6 +57,8 @@ pthread_t pthread_self(void)
 	return t;
 }
 
+#ifdef GIT_WIN_XP_SUPPORT
```

Comment:
@rongjiecomputer given the near zero interest, that I know of, in anyone promising to commit to supporting Windows XP, I am not sure a change like this makes sense to accept. Especially given your prior response:

> I don't have a Windowx XP machine to test, nor do I care to support it.